### ISSUE TITLE
Install fails with "Exit status: 1"

### ISSUE THREAD

**Issue by ttiurani** [clojure-vim/acid.nvim#59]
I'm trying to install the latest acid.nvim but can't for the life of me get the plugin to install.

```
x acid.nvim:
    Exit status: 1
```

I can press R to retry which claims that it works, but still in `:CheckHealth` there is:

```
## Remote Plugins
  - WARNING: "acid.nvim" is not registered.
```

I can't find a way to debug the error any further (not good at all in vimscript).

I have python3 and pynvim installed:

```
## Python 3 provider (optional)
  - INFO: `g:python3_host_prog` is not set.  Searching for python3 in the environment.
  - INFO: Executable: /usr/local/bin/python3
  - INFO: Python version: 3.6.5
  - INFO: pynvim version: 0.3.2
  - OK: Latest pynvim is installed.
```

Other version info:

```
:version
NVIM v0.3.4
Build type: Release
LuaJIT 2.0.5
```

**Comment by hkupty** [comment]
Hi, thanks for the issue.

I'm guessing you installed acid before and now you're trying to install again. Please double-check your plugins folder if it's left there somehow. This looks like a cloning issue.

If not that, make sure that your plugin manager allows cloning git repositories with submodules.

Hopefully it's something as simple as that. :)

**Comment by ttiurani** [comment]
Thanks for the swift response!

I did install acid.nvim before. I'm using the latest vim-plug and have `Plug 'clojure-vim/acid.nvim', { 'do': ':UpdateRemotePlugins' }` which worked with version 1. I also deleted `~/.local/share/nvim` completely and installed everything from scratch, but that didn't help either.

Is there some other place acid.nvim is downloading stuff? Some work dir somewhere?

**Comment by hkupty** [comment]
I'll double check, but there shouldn't be. This is really weird.
You could try cloning acid yourself and using `Plug '/path/to/your/acid.nvim'` to check if it works, so you circumvent any cloning issues.

**Comment by ttiurani** [comment]
Quite the mystery, with:

```
Plug 'file:///my-path-to/acid.nvim', { 'do': ':UpdateRemotePlugins' }
```

I still get the same error.

**Comment by hkupty** [comment]
Oh wow. 

If you don't add the `{ 'do': ..}` part and run that yourself, does it still happen?

**Comment by ttiurani** [comment]
Yup, same thing happens with just `Plug 'file:///my-path-to/acid.nvim'`. If I knew how to debug this I could do that myself, but really don't even understand what vim-plug is doing there. This flashes:

```
Submodule nrepl-python-client ... registerd for path
Cloning into ... nrepl-python-client
```

for a while, but after that something more happens, so it doesn't seem like the cause.

**Comment by hkupty** [comment]
try running `:messages` after you get the error, see if there's something related... I'm not sure if there's any vim-plug debugging that can be turned on.. I'll check that

**Comment by ttiurani** [comment]
Nothing comes up with `:messages`. I tried to google vim-plug logging earlier too, but couldn't get anywhere. What part of this codebase is vim-plug actually running during install?

**Comment by hkupty** [comment]
That's the thing. It shouldn't run anything other than setting up the `rtp` when neovim start. There's no hook or no function should run on `PlugInstall`.

**Comment by hkupty** [comment]
That's why I'm insisting on path or something on vim-plug. All the functions should run on neovim start, after vim-plug's thing:
* `lua/acid/` code should only be loaded on demand, so that's out of the equation.
* `rplugin/python3/` code should run according to the plugin host, which is not there yet because of vim-plug's failure. 
* `plugin/acid.vim` would be called only after setup by vim-plug as well.

Unless there's something related to the `bin` or `githooks` folders, which are only bash scripts for local development. Everything else is asset (markdown or clj code to be injected).

**Comment by hkupty** [comment]
You can try removing both `bin` and `githooks` from your local clone and check if it still fail or not. If it doesn't I'll open a PR to remove those scripts from this repository.

**Comment by ttiurani** [comment]
Under `~/.local/share/nvim/plugged/acid.nvim/rplugin/python3/acid` I have after the install:

```
 zen -> ../../../deps/zen.nvim/src
```

which doesn't point to anywhere. There is only `nrepl-python-client` under deps. How should zen.nvim end up there?

**Comment by ttiurani** [comment]
Took a while, but found out that if I delete the `deps` folder (and commit to the `master` branch), then `PlugInstall` works. Deleting `githooks` and  `bin` doesn't do the trick.

Found it: it was my global git hook taken from http://stackoverflow.com/a/40450106 that failed. There's a bug there somewhere I need to find.

Sorry for the trouble and thanks for your work!

**Comment by hkupty** [comment]
I'm glad that the issue is fixed now.

Just make sure that you have the dependency from deps otherwise acid won't be able to connect to the nrepl.

If you still have any issues, please don't refrain from opening another issue.

Best,
Henry
